FundacionParaguaya/MentorApp
# Bugfix/979 pdf print

**Describe the changes**
Added translation in Spanish 
Added priorities table 

**To Test**
1 Create survey 
2 Create 2 priorities, create 2 achievements,
3 Change language to Spanish
4 Generate PDF 
5 Make sure you see icons for priorities and achievements, table for priorities, translation in spanish
